newdle/client/src/components/NewdleTitle.js

```diff
@@ -46,8 +45,9 @@ export default function NewdleTitle({
                 trigger={
                   <Button
                     icon
-                    active={!isSummary}
-                    onClick={() => (isSummary ? history.push(answerURL) : null)}
+                    active={!isSummaryRoute}
```

Comment:
We need to add another `!isEditRoute` check here, since we need the following logic:

- on answer route (hard to match here): answer active, summary clickable
- on summary route: answer clickable, summary active
- on edit route: both clickable, none active

Reply:
Good point, we also desperately need an alternative routing solution, I'm adding a TODO on that to prevent adding more changes here.